utt/privacy-wallet-service/src/Wallet.cpp

```diff
@@ -22,11 +22,10 @@ using namespace utt::client::utils::crypto;
 Wallet::Wallet(std::string userId,
                const std::string& private_key,
                const std::string& public_key,
-               const std::string& storage_path,
+               const std::shared_ptr<utt::client::IStorage> storage,
```

Comment:
why did u need this change?

Reply:
IStorage object (FileBasedUserStorage) is constructed in PrivacyService and injected to Wallet to enable calling storage functions from PrivacyService